From f83b52737e321005959497d8e8f59f8aaedc9048 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Tue, 26 Jan 2016 14:34:01 +0100
Subject: [PATCH] Do not process subforms with PMA_MINIMUM_COMMON
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

In such case needed infrastructure is not loaded, so related code won't
work anyway.

Signed-off-by: Michal Čihař <michal@cihar.com>
---
 libraries/common.inc.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libraries/common.inc.php b/libraries/common.inc.php
index 6686cebb741..fd85814f943 100644
--- a/libraries/common.inc.php
+++ b/libraries/common.inc.php
@@ -231,7 +231,7 @@
  * so we now check if a subform is submitted
  */
 $__redirect = null;
-if (isset($_POST['usesubform'])) {
+if (isset($_POST['usesubform']) && ! defined('PMA_MINIMUM_COMMON')) {
     // if a subform is present and should be used
     // the rest of the form is deprecated
     $subform_id = key($_POST['usesubform']);
